api/server/sdk/server.go

```diff
@@ -481,7 +479,7 @@ func (s *sdkGrpcServer) useAlert(a alerts.FilterDeleter) {
 // Accessors
 func (s *sdkGrpcServer) driver(ctx context.Context) volume.VolumeDriver {
 	driverName := grpcserver.GetMetadataValueFromKey(ctx, ContextDriverKey)
-	if driverName == "" {
+	if s.driverHandlers[driverName] == nil {
```

Comment:
This works, but normally it is written as follows:

```go
if handler, ok := s.driverHandlers[driverName]; ok {
    return handler
} else {
    return s.driverHandlers[DefaultDriverName]
}
```

Reply:
Also just realized that the code as it is may not be what the caller wanted. Since they asked for a driver, and that driver, maybe, had been initialized to `nil` by the caller to the SDK, then instead the caller would be getting the Default driver. With the snippet in the comment above, the caller would be getting whatever value was placed in the driver handler for the driver requested. I'm assuming the latter is what we want, right?

Reply:
@lpabon You are right!
@jrivera-px Lets modify it to what Luis has suggested.

Reply:
Yeah definitely ... will do it  now..